feathr-example/bureau_bureau-balance.ipynb

```diff
@@ -0,0 +1,1224 @@
+{
```

Comment:
Can you standardise the names of the notebooks? I see "namea`_`nameb.ipynb" and "namea`-`nameb.ipynb", etc.

Reply:
Standardize it to this. 
![image](https://user-images.githubusercontent.com/308396/191151434-ed3c6f6b-7391-475d-828b-efb04983a9ae.png)
the filename would be <CSV_NAME>-and-<ANOTHER_CSV_NAME>